lib/tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "sourceMap": true,
     "importHelpers": true,
     "skipLibCheck": true,
+    "declaration": true,
```

Comment:
I think `declaration` does not compatible with `allowJs` and `isolatedModules`. Is rollup using `tsconfig.rollup.json`?

Reply:
I've removed it in b4c8c85, I had forgotted that I'd added it